lib/rubocop/cop/lint/safe_navigation_chain.rb

```diff
@@ -52,6 +54,13 @@ def on_send(node)
 
         private
 
+        def require_safe_navigation?(node)
+          parent = node.parent
+          return true unless parent&.and_type?
+
+          parent.rhs != node || parent.lhs.receiver != parent.rhs.receiver
```

Comment:
Shouldn't we also check if `parent.lhs` is a `send` node before calling `.receiver`?

Reply:
As far as I have verified, it seems there are no cases where this does not respond to `RuboCop::AST::Node#receiver`.

Reply:
Ah, nice, was thinking `receiver` is only for send nodes.